src/networking/testSimulator/Client.java

```diff
@@ -0,0 +1,75 @@
+package networking.testSimulator;
+
+import networking.ICommunicator;
+import networking.utility.ClientInfo;
+
+
+/**
+ *  This class represents a dummy client which has a processingManager and contentManager
+ *  @author Dasari Sravan Kumar
+ */
+public class Client extends Thread {
+	
+	ICommunicator communicator=null;
+	ClientInfo srcClient=null;
+	ClientInfo destClient=null;
+	Message  input = null;
+	Message output = null; 
+	int numMessages;
+	int msgLength;
+	
+	
+	Stopper stopper = null;
+ 	
+	
+	public Client(
+					  ICommunicator communicator,
+					  ClientInfo client1,
+					  ClientInfo client2,
+					  Message input,
+					  Message output,
+					  int numMessages,
+					  int msgLength,
+					  Stopper stopper
```

Comment:
Fix indentation to keep it consistent with other files.
If you don't see the difference on your local editor, check this on the GitHub file viewer, you will understand the difference.